### ISSUE TITLE
Category List Boolean fields may not be correctly parsed when importing files.

### ISSUE THREAD

**Issue by elmatjon** [shanewalters/qif#7]
Whenever I import a list of transaction categories the Boolean fields, i.e. IncomeCategory, ExpenseCategory, and TaxRelated, are always false, regardless of their values in the .qif file.

It looks like the API is expecting explicit true/false string values in the .qif file, but quicken treats it more like an option flag when exporting data, i.e. the existence of the field id code (e.g. T, I, and E) is sufficient to indicate TRUE, and its absence indicates FALSE.  Or at least that's how I'm interpreting the spec, since it fits the .qif files I' trying to import.

Here's an excerpt from my .qif file:

```
!Type:Cat
NBonus
DBonus Income
T
R7360
I
^
NClothing
DClothing
E
^
NEntertainment
DEntertainment
E
^
NMedical:Doctor
DDoctors, Dentists, & Hospitals
T
R7744
E
^
!Clear:AutoSwitch
```

So I should have one income and three expense categories, of which one of each is tax-related.

But if I import and immediately export, the above becomes: 

```
!Type:Cat
B0
NBonus
DBonus Income
EFalse
IFalse
TFalse
R7360
^
B0
NClothing
DClothing
EFalse
IFalse
TFalse
^
B0
NEntertainment
DEntertainment
EFalse
IFalse
TFalse
^
B0
NMedical:Doctor
DDoctors, Dentists, & Hospitals
EFalse
IFalse
TFalse
R7744
^
```

Now all the categories are neither income nor expense, which doesn't match what was exported from Quicken.

I don't know if the .qif spec has changed over time, but my files are from Quicken 2000, so it's possible that more modern versions are using the explicit true/false values instead of just the field indicators.

I'm in the process of modifying the API to work with my import files.  I'd be happy to submit a PR when I'm finished, but I don't know if the API should look just for the presence/absence of field indicators for income/expense/tax-related and nothing else, or if it should also parse explicit true/false strings.

For example, should an Income category be represented like this:

`I`

and not this:

`ITrue`

Or should both options be acceptable?

**Comment by shanewalters** [comment]
Great catch. Reading the spec seems to indicate mere presence should dictate truth. It also says absence of either I or E should default to expense. I'll wire up an update and get this fixed.

**Comment by shanewalters** [comment]
This should have been corrected in commit 19de1c97201ae0f94dd6d8e7d982573312061330.
